HHS/OPRE-OPS
# corrects Alert message when sending to Agreement to review

## What changed

corrects Alert message when sending to Agreement to review

## Issue

~~_Add link to issue here_~~

## How to test

1. send an Agreement to review
2. see correct alert msg
